src/snowflake/snowpark/modin/plugin/_internal/utils.py

```diff
@@ -273,7 +273,9 @@ def _create_read_only_table(
     readonly_table_name = (
         f"{random_name_for_temp_object(TempObjectType.TABLE)}{READ_ONLY_TABLE_SUFFIX}"
     )
-    use_scoped_temp_table = session._use_scoped_temp_objects
+    use_scoped_temp_table = (
+        session._use_scoped_temp_objects and pd.configs.enable_scoped_snapshot
```

Comment:
Does this behavior make sense if `session._use_scoped_temp_objects` is False and `pd.configs.enable_scoped_snapshot` is True?

Would it be reasonable to instead have `pd.configs.enable_scoped_snapshot` just be an alias for `session._use_scoped_temp_objects`, so that if a user does `pd.configs.enable_scoped_snapshot = False`, it will delegate the set to `session._use_scoped_temp_objects` instead? I'm worried about having two different configuration variables that seem to address the same behavior.